### ISSUE TITLE
update runner tests to send interrupt

### ISSUE THREAD

**Issue by charlierudolph** [Originate/exosphere#444]
This will trigger the shutdown which we can wait for to complete. Hopefully cause our tests to run more reliably since we properly shut down all containers. With this I think we could remove the step of kill all containers in our after scenario. 

I think to start we should just use the "And I stop all running processes" in more places and update it to send the signal and wait instead of killing the process.

@hugobho @cheahuychou @mcharawi thoughts?

**Comment by hugoesthere** [comment]
Yes this would work, but only for Docker processes booted up by `exo run` right? Are there any other Docker processes run in tests that don't use `exo run`?
